Header header
string address
uint16 srcPort
uint8[] data
